db/dbformat.h

```diff
@@ -327,21 +327,42 @@ inline int InternalKeyComparator::Compare(const InternalKey& a,
   return Compare(a.Encode(), b.Encode());
 }
 
+inline Status ReturnCorruptKeyInfo(ParsedInternalKey* ikey, bool log_err_key) {
+  std::string msg("Corrupted Key. ");
+
+  if (log_err_key) {
+    // print key in Hex
+    msg.append("Key=" + ikey->user_key.ToString(true) + ", ");
```

Comment:
I know it is copy&paste code but since we are generalizing the code, can we consolidate the logic with ParsedInternalKey::DebugString()?

Reply:
Thanks for your review Siying. I missed noticing ParsedInternalKey::DebugString() - this is the right/necessary place to add this logic. Will fix this and other feedbacks and repost